projects/addon-doc/src/components/tab-content/tab-content.template.html

```diff
@@ -0,0 +1,17 @@
+<div @tuiFadeIn>
```

Comment:
If we use it for animation binding only, I'm pretty sure we can bind it to host.

Reply:
Somehow I am getting strange behavior in this case

```ts
    host: {
        '[@tuiFadeIn]': 'true',
    },
```

```css
:host {
  display: block;
}
```

https://user-images.githubusercontent.com/12021443/149759026-61aa69d0-2aa9-4442-83c4-48dcb4099424.mov